src/core/client/admin/helpers/markHTMLNode.ts

```diff
@@ -1,7 +1,13 @@
+const LENGTH_CUTOFF = 500;
```

Comment:
If you go any higher than this with a specific high profile client's banned/suspect word list, you will get massive slowdowns when running the regex powered `text.split(...)` over it...